src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -210,34 +201,61 @@ internal bool NullableSemanticAnalysisEnabled
         /// </summary>
         internal bool IsPeVerifyCompatEnabled => LanguageVersion < LanguageVersion.CSharp7_2 || Feature("peverify-compat") != null;
 
-        internal bool ShouldRunNullableWalker
+        /// <summary>
+        /// Returns true if nullable analysis should be run in this compilation.
```

Comment:
I feel like "should be run" is a bit ambiguous here an I think that extends to the property name as well. I'd phrasing like "is nullable analysis enabled / allowed".

Reply:
Renamed to IsNullableAnalysisEnabled`.

---
In reply to: [540610442](https://github.com/dotnet/roslyn/pull/49876#discussion_r540610442) [](ancestors = 540610442)